### DISCUSSION TITLE
Missing samples for  desktop / webassembly

### DISCUSSION THREAD

**Discussion by ddobric** [dotnet/maui#193] [General]
Looking for samples and concepts that demonstrate how to build desktop applications with MAUI. 
For example a MAUI application compiled as webassembly, which can be executed in browser, installed as progressive app etc.

Thanks in advance.

**Comment by MarioGK** [comment]
A "quick start" or "getting started guide" would be awesome as well.

**Comment by GiampaoloGabba** [comment]
Unfortunately webassembly is not a supported platform (at least for the first version scheduled for the end of 2021) :(
I doubt we will see any "official" example for a long time, maybe we will see some community contributions (for example, UNO has developed webassembly renderers for Xamarin.Forms)
